test/e2e/app-dir/metadata/next.config.js

```diff
@@ -1 +1,4 @@
-module.exports = {}
+/** @type {import('next').NextConfig} */
+const config = {}
```

Comment:
We'd need `experimental: {ppr: 'incremental'}` for it to repro, or we could add it to `test/e2e/app-dir/ppr-metadata-streaming` instead.

Reply:
To be fair, I didn't check if it ran with PPR. I just hoped it would because that feels like an important feature to test with PPR.